packages/types/src/customer/common.ts

```diff
@@ -1,6 +1,25 @@
 import { AddressDTO } from "../address"
 
-export interface CustomerDTO {
+export type CustomerDTO = {
+  id: string
+  email: string
+  default_billing_address_id?: string | null
+  default_shipping_address_id?: string | null
+  company_name?: string | null
+  first_name?: string | null
+  last_name?: string | null
+  default_billing_address?: AddressDTO
+  default_shipping_address?: AddressDTO
+  addresses?: AddressDTO[]
+  phone?: string | null
+  groups?: { id: string }[]
+  metadata?: Record<string, unknown>
+  deleted_at?: Date | string
+  created_at?: Date | string
+  updated_at?: Date | string
```

Comment:
suggestion: we could now remove `?` since they have a default initializer. to be validated cc @olivermrbl

Reply:
We will figure that out at a later stage

Reply:
got it, sounds good to me, just wanted to raise that point but fine to keep it for later